### DISCUSSION TITLE
How to un-minimize the window?

### DISCUSSION THREAD

**Discussion by aurexav** [emilk/egui#4635] [Q&A]
When the app loses focus, it will execute the following code, minimizing the window to the dock:

```rs
ctx.send_viewport_cmd(ViewportCommand::Minimized(true));
```

However, I want it to pop up when a user presses a hotkey. How can I achieve this?

I tried these methods but they didn't work.

```rs
ctx.send_viewport_cmd(ViewportCommand::Minimized(false));
ctx.send_viewport_cmd(ViewportCommand::Maximized(true));
ctx.send_viewport_cmd(ViewportCommand::Focus);
```

**Comment by phoglund** [comment]
See what I wrote in https://github.com/emilk/egui/issues/3655, if you're win only I posted a hack you can use
